src/fu-engine.c

```diff
@@ -2057,8 +2057,25 @@ fu_engine_get_result_from_app (FuEngine *self, AsApp *app, GError **error)
 
 	/* verify trust */
 	release = as_app_get_release_default (app);
-	if (!fu_keyring_get_release_trust_flags (release, &trust_flags, error))
-		return NULL;
+	g_autoptr(GError) error_local = NULL;
```

Comment:
This is causing a CI failure mixing declarations with code.  This line should be moved up above to function declarations (around line 2016)